output/output.json

```diff
@@ -0,0 +1,150 @@
+{
+    "input_1": {
```

Comment:
The complete filename was requested as the upper-level node `input_1.png` rather than `input_1`